js/endingletters.js

```diff
@@ -0,0 +1,25 @@
+// source: intjson.proto
+/**
+ * @fileoverview
+ * @enhanceable
+ * @suppress {missingRequire} reports error on implicit type usages.
+ * @suppress {messageConventions} JS Compiler reports an error if a variable or
+ *     field starts with 'MSG_' and isn't a translatable message.
+ * @public
+ */
+// GENERATED CODE -- DO NOT EDIT!
+/* eslint-disable */
+// @ts-nocheck
+
+goog.provide('proto.intjson.EndingLetters');
```

Comment:
We don't use this stack in TypeScript GAPICs, we use protobuf.js instead. @summer-ji-eng could point you to the right direction.